types/index.d.ts

```
@@ -27,3 +28,128 @@ interface UserInfoCard {
+  id: string;
+  title: string;
+  description: string;
+  component: React.ReactNode;
+}
+
+interface MultiStepFormProps {
+  steps: Step[];
+  onComplete: (id: string) => void;
+}
+
+interface CTAStep {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)